### ISSUE TITLE
How Is the 25% Test Split Selected?

### ISSUE THREAD

**Issue by Khola-naseem** [ucsk/ppbotsort#2]
Thank you for the great work on this project! I have a quick question regarding the data split:
Is the first 25% of the data used for testing, or is it the last 25%?

**Comment by ucsk** [comment]
> Thank you for the great work on this project! I have a quick question regarding the data split: Is the first 25% of the data used for testing, or is it the last 25%?

The first 25% is used for testing.

```python
split_len = int(len(img_name_list) * 0.25)
train_list = img_name_list[split_len:]
val_list = img_name_list[:split_len]
```

**Comment by Khola-naseem** [comment]
Thank you for your response. However, I noticed in the repository  you mentioned the opposite: "The CTMC dataset1 is divided into the first 75% (training set) and the second 25% (validation set)."

Could you please clarify this?

**Comment by ucsk** [comment]
Hi, the description of this repository may be ambiguous, but in terms of code, the data split method is the same as the [previous method](https://github.com/ucsk/ppbytetrack/blob/main/README.md?plain=1#L7).

When comparing detector performance, it is also necessary to ensure dataset consistency.
